scripts/cnv_wdl/somatic/cnv_oncotate.wdl

```diff
@@ -0,0 +1,51 @@
+workflow CNVOncotateCalledSegments {
+    File called_file
+    Int? mem
+    String? oncotator_docker
+    String? oncotator_output_type
+    String? additional_args
+
+    call OncotateSegments {
+        input:
+            called_file=called_file,
+            mem=mem,
+            oncotator_docker=oncotator_docker,
+            oncotator_output_type=oncotator_output_type,
+            additional_args=additional_args
+    }
+
+    output {
+        File oncotated_called_file = OncotateSegments.oncotated_called_file
+    }
+}
+
+task OncotateSegments {
+    File called_file
+    String? oncotator_output_type
+    String? additional_args
+    String basename_called_file = basename(called_file)
+
+    # Runtime parameters
+    Int? mem
+    String? oncotator_docker
+    Int? preemptible_attempts
+    Int? disk_space_gb
+
+    command <<<
+        set -e
+        /root/oncotator_venv/bin/oncotator --db-dir /root/onco_dbdir/ -c /root/tx_exact_uniprot_matches.AKT1_CRLF2_FGFR1.txt \
+          -u file:///root/onco_cache/ -r -v ${called_file} ${basename_called_file}.per_segment.oncotated.txt hg19 \
+          -i SEG_FILE -o ${default="SIMPLE_TSV" oncotator_output_type} ${default=" " additional_args}
```

Comment:
Use an empty `""` in `${default=" " additional_args}`, if you want to be absolutely consistent...